AsyncDisplayKit/Details/ASPINRemoteImageDownloader.m

```diff
@@ -85,7 +85,23 @@ - (PINRemoteImageManager *)sharedPINRemoteImageManager
   static PINRemoteImageManager *sharedPINRemoteImageManager = nil;
   static dispatch_once_t onceToken;
   dispatch_once(&onceToken, ^{
+  
 #if PIN_ANIMATED_AVAILABLE
+    // Check that Carthage users have linked both PINRemoteImage & PINCache by testing for one file each
+    if (!(NSClassFromString(@"PINRemoteImageManager.m"))) {
+        NSException *e = [NSException
+                          exceptionWithName:@"FrameworkSetupException"
+                          reason:@"Missing the path to the PINRemoteImage framework."
+                          userInfo:nil];
+        @throw e;
+    }
+    if (!(NSClassFromString(@"PINCache.m"))) {
```

Comment:
I guess same for this case as above, you should remove the `.m` at the end of the class